blocks/library/quote/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { html, query } = hpq;
+
+registerBlock( 'core/quote', {
+	title: wp.i18n.__( 'Quote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: ( node ) => query( 'blockquote > p', html() )( node ).join(),
```

Comment:
Good point! What about adding a `div` wrapper inside the blockquote to separate the footer from the content properly. This will also resolve the `dangerouslySetInnerHTML` issue?